components/automate-gateway/handler/iam/v2/tokens/tokens.go

```diff
@@ -116,16 +116,6 @@ func (s *Server) ListTokens(
 	return &pb_resp.ListTokensResp{Tokens: tokens}, nil
 }
 
-// ResetAllTokenProjects removes all projects from every token
-func (s *Server) ResetAllTokenProjects(
```

Comment:
yeah this is fine to delete. it was only used to clear out any straggling `default` project references